src/app/pages/loading-waiting-states/contextual-spinners/contextual-spinners.component.scss

```diff
@@ -1,19 +1,101 @@
 @import '~@pxblue/colors/palette.scss';
 
-.extended-fab-button {
-    width: 180px;
-    height: 56px;
-    padding: 0 26px;
-    border-radius: 29px;
-    transition-property: unset;
-    transition-duration: 1s;
-
-    .mat-icon {
-        margin-right: 8px;
+section.mat-typography {
+    padding: 16px 24px;
+    .example-section {
+        margin-bottom: 16px;
+    }
+    .show-text {
+        .text-container {
+            opacity: 1;
+            visibility: visible;
+            -webkit-transition: opacity 200ms, visibility 200ms;
+            transition: opacity 200ms, visibility 200ms;
+        }
+        .spinner-container {
+            position: absolute;
+            opacity: 0;
+            visibility: hidden;
+            -webkit-transition: opacity 200ms, visibility 200ms;
+            transition: opacity 200ms, visibility 200ms;
+            margin-top: -28px;
+            margin-left: 10px;
+        }
     }
 
-    .primary-spinner {
-        display: none;
+    .raised-button-example {
+        height: 36px;
+        .text-container {
+            position: absolute;
+            opacity: 0;
+            visibility: hidden;
+            -webkit-transition: opacity 200ms, visibility 200ms;
+            transition: opacity 200ms, visibility 200ms;
+            margin-top: -8px;
+            margin-left: 4px;
+        }
+        .spinner-container {
+            opacity: 1;
+            visibility: visible;
+            -webkit-transition: opacity 200ms, visibility 200ms;
+            transition: opacity 200ms, visibility 200ms;
+            margin: 0 12px;
+        }
+    }
+
+    .extended-fab-button {
+        width: 180px;
+        height: 56px;
+        border-radius: 29px;
+        transition: width 300ms;
+    }
+
+    .show-icon {
+        .icon-container {
+            margin-right: 16px;
```

Comment:
Should be: 
![image](https://user-images.githubusercontent.com/8997218/123335556-a3d98000-d512-11eb-8c4c-d29e339d7fa0.png)

As a comparison, design looks like this:
![image](https://user-images.githubusercontent.com/8997218/123335588-b05dd880-d512-11eb-8c5d-61066b05d962.png)

Reply:
Fixed width